src/app/content/components/TableOfContents/index.tsx

```diff
@@ -136,6 +136,22 @@ function TocNode({
   );
 }
 
+function maybeAriaLabel(page: LinkedArchiveTreeNode) {
+  const [num, titleText] = splitTitleParts(page.title);
+
+  if (num) {
+    return {};
+  }
+
+  const [parentNum, parentTitleText] = splitTitleParts(page.parent!.title);
```

Comment:
Can't parent still be undefined here? It seems like a guard could be added to the early return above and then you'd have a narrowed type

Reply:
The test data doesn't have any with null parents. The prospect of having to add to the test data made me look at likely TOC items like "Index", which has as its parent the book itself. I don't think there are any TOC entries that don't have a parent.

Reply:
Does that mean the type should be changed? Maybe @TomWoodward can chime in, I don't know these types well enough to feel confident about asserting the data will always be there.

Reply:
i think in real life the only thing in the linked section that won't have a parent is the book node. its possible the types could be updated to make parent required

Reply:
Turns out the type I was using was a union, and it only needs to be the one where `parent` is required.

Reply:
Ah good find